### ISSUE TITLE
Sleep/Wake issues Big Sur

### ISSUE THREAD

**Issue by ekristen** [docker/for-mac#5489]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior

MacBook Pro should sleep/wake properly.

### Actual behavior

When Docker for Mac is running, once the computer enters the sleep cycle, it can take up to 2-3 minutes for the system to wake back up. 

### Information

  - macOS Version: 11.2.1
  - Docker Desktop Version: 3.2.2

It is reproducible for me 100% of the time, happened on a fresh install on a brand new macbook pro running big sur. Disabling Docker for Mac allows the laptop sleep/wake process to work as expected.

### Steps to reproduce the behavior

  1. MacBook Pro 2019
  2. Docker for Mac
  3. Allow system to sleep
  4. Attempt to wake

**Comment by djs55** [comment]
@ekristen thanks for your report. As an experiment could you go to: whale menu -> Preferences... -> Experimental Features and enable "Use new virtualization framework"? This will switch from [hyperkit](https://github.com/moby/hyperkit] to Apple's new hypervisor which might help with sleep/wake issues. Let me know if it helps or not!

**Comment by ekristen** [comment]
@djs55 thanks for the suggestion, I am trying that as part of trying to figure out what is going one. I thought I had narrowed this down to Docker, but then it started happening again two days ago with Docker for Mac disabled :( 

I think that it was circumstantial that it "appeared" to be docker. 

I think I narrowed it down to the MBP switching between the graphics card, it seems whenever the discrete graphics card is active, it can lead to a wake from sleep taking 2-3 minutes. Supposedly was a bug fixed by Big Sur, but still seeing the issues. 

I'm still running tests, but I think for now I can say I no longer think it is docker for mac, which is good. 

I'm going to close this for now, if that changes I'll double back around. Appreciate the response.
